tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -24431,7 +24432,1322 @@ TEST_F(EnergyPlusFixture, VRF_MultispeedFan_Test)
     EXPECT_LT(abs(QZnReq - SysOutputProvided), 1.0);
     EXPECT_EQ(state->dataHVACVarRefFlow->VRFTU[0].SpeedNum, 1);
     EXPECT_NEAR(state->dataHVACVarRefFlow->VRFTU[0].SpeedRatio, 0.0, 0.00001);
-    EXPECT_NEAR(state->dataHVACVarRefFlow->VRFTU[0].CycRatio, 0.713222413, 0.00001);
+    EXPECT_NEAR(state->dataHVACVarRefFlow->VRFTU[0].CycRatio, 0.713204045, 0.00001); // was 0.713222413
+}
+
+TEST_F(EnergyPlusFixture, VRF_MultispeedFan_Test_HardSized)
```

Comment:
New unit test with hard-sized inputs for airflow, capacity and SHR that are duplicate of results from above unit test.